hubblestack_nebula/hubblestack_nebula_win_queries.yaml

```diff
@@ -7,7 +7,7 @@ fifteen_min:
     query: SELECT t.unix_time AS query_time, h.md5 AS md5, p.pid AS process_id, p.name AS process, ltrim(address, ':f') AS address, port AS dest_port, p.path AS file_path, p.cmdline, p.on_disk, p.root, p.parent, pp.name AS parent_process, CASE lp.protocol WHEN 6 THEN 'tcp' WHEN 17 THEN 'udp' ELSE lp.protocol END as transport FROM listening_ports AS lp LEFT JOIN processes AS p ON lp.pid=p.pid LEFT JOIN processes AS pp ON p.parent=pp.pid LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path=p.path;
 hour:
   - query_name: win_drivers
-    query: SELECT t.unix_time AS query_time, d.*, h.md5 AS md5, h.sha1 AS sha1, h.sha256 AS sha256 FROM drivers AS d LEFT JOIN time AS t LEFT JOIN hash AS h ON h.path = replace(d.path,'\SystemRoot','C:\Windows');
```

Comment:
This query is failing on windows with the error:
```Error: no such column: d.path```

In osquery 2.8, 'path' column of drivers table has been replaced by 'image' column.
So, a change in this query is required to make it work with osquery 2.8

Moreover, in osquery 2.8, in d.image column, all paths are starting from 'C:\Windows' ( instead of '\SystemRoot' ). So there is no need to replace '\SystemRoot' with 'C:\Windows'.